packages/databricks-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# Release: v0.3.0
+
+## packages/databricks-vscode
+
+## 0.3.0 (2023-02-20)
+
+-   ⚠️ Breaking change: Don't list exsiting Repos when selecting a sync destination. This prevents users from accidentally overwriting code in existing
```

Comment:
```suggestion
-   ⚠️ Breaking change: Don't list existing Repos when selecting a sync destination. This prevents users from accidentally overwriting code in existing 
```

Is "breaking change" an overstatement? This looks very scary right now with the exclamation sign, but it actually doesn't seem like something users should really be worried about all that much? And if it is a breaking change could you rewrite it so it's immediately clear that almost all users aren't affected?